learning-another-language/hello-world.md

```diff
@@ -0,0 +1,279 @@
+# Hello, World!
+
+## Introduction
+
+A well-known exercise to accomplish when learning another programming language is "Hello, World!" We'll know we're successful if we see the text `Hello, World!` appear somewhere after running the code.
+
+## Running Code in the REPL
+
+To run JavaScript code, we need to consider its _runtime environment_. A **runtime environment** is a set of technologies that determine _how_ to execute the code.
+
+Some examples of runtime environments that can run JavaScript:
+
+- The runtime environment of a web browser like Mozilla Firefox and Google Chrome
+- [Node.js](https://nodejs.org/en/)
+- REPLs like [replit.com](https://replit.com/)
+
+For this portion of the curriculum, we will use [replit.com](https://replit.com/).
+
+The following steps will let us create and run JavaScript programs with ease:
+
+1. Create an account on [replit.com](https://replit.com/) and log-in
+1. Visit [the Hello, World! REPL](https://replit.com/@adacore/hello-world#index.js)
+
```

Comment:
add an image and explanation for the preview we see for a repl that we aren't invited to edit
```suggestion

When we open a REPL that we don't own, we will see a page with the project name, and a play button to run the project. We can toggle the view to show the code. We can also make a copy of the project in our own account by clicking Fork.

![JavaScript readonly REPL. The browser displays the name of the project, hello-world, and a large green button to run the project. A toggle labelled Code and button labelled Fork are also visible.](../assets/learning-a-new-language_hello-world_repl-unowned.png)

Let's Fork this project and see what that lets us do!

```